packages/test-interface-criteria/src/criteria/internal/InternalTestOpportunityBookable.js

```diff
@@ -12,23 +12,16 @@ const { getRemainingCapacity, createCriteria, mustBeWithinBookingWindow } = requ
  * @type {OpportunityConstraint}
  */
 function remainingCapacityMustBeAtLeastTwo(opportunity) {
-  return getRemainingCapacity(opportunity) > 1;
+  return getRemainingCapacity(opportunity) > (hasCapacityLimitOfOne(opportunity) ? 0 : 1);
 }
```

Comment:
Maybe worth adding a comment that 2 is needed in other cases because the multiple OrderItem tests use 2 of the same item.

I feel that this function and the opportunityReuseKey logic should be derived from the same foundation in some way (e.g. a config?). Otherwise, it seems like it'd be easy to change one without realising the other needs changing (e.g. "maybe we should have 3 items with the same reuse key")

However, that would require some thinking time and I understand this is urgent

Reply:
Yes great point, have added an issue for this to be picked up in future: https://github.com/openactive/openactive-test-suite/issues/247